frontend/src/metabase/query_builder/components/expressions/Editor/Editor.tsx

```diff
@@ -4,48 +4,50 @@ import cx from "classnames";
 import { useCallback, useMemo, useRef, useState } from "react";
 import { useMount } from "react-use";
 import { t } from "ttag";
+import _ from "underscore";
 
 import { useSelector } from "metabase/lib/redux";
 import { getMetadata } from "metabase/selectors/metadata";
-import { Box } from "metabase/ui";
+import { Box, Button, Flex, Icon } from "metabase/ui";
 import * as Lib from "metabase-lib";
-import { format, isExpression } from "metabase-lib/v1/expressions";
-import type { Shortcut } from "metabase-lib/v1/expressions/complete";
+import { format } from "metabase-lib/v1/expressions";
 import { tokenAtPos } from "metabase-lib/v1/expressions/complete/util";
-import { diagnose } from "metabase-lib/v1/expressions/diagnostics";
-import { processSource } from "metabase-lib/v1/expressions/process";
 import { TOKEN } from "metabase-lib/v1/expressions/tokenizer";
 import type { ErrorWithMessage } from "metabase-lib/v1/expressions/types";
 import type Metadata from "metabase-lib/v1/metadata/Metadata";
-import type { Expression } from "metabase-types/api";
 
 import type { ClauseType, StartRule } from "../types";
 
 import S from "./Editor.module.css";
+import { Errors } from "./Errors";
```

Comment:
Custom column name input does not have a focus effect

Reply:
@AlessioLaiso What should the focus effect be here? We didn't add one in the designs.